tip-wlan/charts/wlan-port-forwarding-gateway-service/values.yaml

```diff
@@ -170,3 +170,5 @@ env:
   pfGateway:
     service: wlan-port-forwarding-gateway-service
     port: 7070
+    externallyVisibleHost: pfg.example.org
```

Comment:
Though the properties are fine here but in general I tend to keep them out of the "env" since we are controlling values under "env" from the template defined under common/templates/env.tpl file.
I would recommend that we do something similar to what we did in opensync-gw-cloud - see https://github.com/Telecominfraproject/wlan-cloud-helm/blob/master/tip-wlan/charts/opensync-gw-cloud/values.yaml#L120-L122

Reply:
Thank you Rahul, I will make this change. And the Values in deployment.yaml would be accessible as .Values.pfGateway.externallyVisiblePort    instead of  .Values.env.pfGateway.externallyVisiblePort

Reply:
I would remove even pfGateway. So something as follows:
```
externallyVisible:
    host:
    port:
```
since we already have `wlan-port-forwarding-gateway-service` under which these properties are defined, so no need to repeat. What do you think?

Reply:
Yes, even I think pfGateway is not necessary.  I will make these changes. Thank you very much!